### DISCUSSION TITLE
Problema aggiornamento

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#315] [Richieste di aiuto]
Buonasera, dopo aver effettuato l'aggiornamento del registro all'ultima versione da voi rilasciata , mi compare l'errore 500.

**Comment by trinko** [comment]
Parli della versione 1.4.5? 
Vedi nei log se ci sono dettagli sull'errore, altrimenti non sono in grado di fare niente.

**Comment by rodolfo024** [comment]
[](url
[app_prod-2022-08-23.log](https://github.com/iisgiua/giuaschool/files/9415398/app_prod-2022-08-23.log)
)

**Comment by trinko** [comment]
Vedo che hai cercato di installare le modifiche **successive** alla 1.4.5: non farlo, la procedura di installazione non è ancora pronta.
Attendi il rilascio della 1.5.0 e poi farai l'installazione di quella.
